### ISSUE TITLE
Bootstrap Frontend Update

### ISSUE THREAD

**Issue by cycomachead** [snap-cloud/snapCloud#364]
Here's a checklist of all files that need updating, and their progress.

All relative to `views/`, files that have `_bs` mean I'm keeping both versions around for a little while. (Either it's a partial, or for testing.)

- [x] /dialogs/collect.etlua
- [x] /dialogs/compose_email.etlua
- [x] /dialogs/confirm_delete.etlua
- [x] /dialogs/delete_user.etlua
- [x] /dialogs/embed.etlua
- [x] /dialogs/feature_carousel.etlua
- [x] /dialogs/flag_reason.etlua

**layout** (reorganized in #359)
- [x] /layout/delayed_scripts.etlua
- [x] /layout/footer_bs.etlua
- [x] /layout/head_bs.etlua
- [x] /layout/navigation_bar_bs.etlua

**partials**
- [x] /partials/carousel_paginator_bs.etlua
- [x] /partials/carousel_paginator.etlua
- [x] /partials/collection_buttons.etlua
- [x] /partials/collection_dates.etlua
- [x] /partials/collection_editors.etlua
- [x] /partials/collection_info.etlua
- [x] /partials/collection_thumbnail.etlua
- [x] /partials/collection.etlua
- [x] /partials/flag_list.etlua
- [x] /partials/grid_filters_bs.etlua
- [x] /partials/grid_filters.etlua
- [x] /partials/grid_paginator_bs.etlua
- [x] /partials/grid_paginator.etlua
- [x] /partials/grid_search_bs.etlua
- [x] /partials/grid_search.etlua
- [x] /partials/profile_bs.etlua
- [x] /partials/profile.etlua
- [x] /partials/project_bs.etlua
- [x] /partials/project_buttons.etlua
- [x] /partials/project_details.etlua
- [x] /partials/project.etlua
- [x] /partials/slideshow_bs.etlua
- [x] /partials/slideshow.etlua
- [x] /partials/user.etlua

**static** (Reorganized in #359 -- most of these needed no or minimal changes.)
- [x] /static/about.etlua
- [x] /static/bjc.etlua
- [x] /static/blog.etlua
- [x] /static/coc.etlua
- [x] /static/contact.etlua
- [x] /static/credits.etlua
- [x] /static/dmca.etlua
- [x] /static/extensions.etlua
- [x] /static/materials.etlua
- [x] /static/mirrors.etlua
- [x] /static/offline.etlua
- [x] /static/partners.etlua
- [x] /static/privacy.etlua
- [x] /static/requirements.etlua
- [x] /static/research.etlua
- [x] /static/snapinator.etlua
- [x] /static/snapp.etlua
- [x] /static/source.etlua
- [x] /static/tos.etlua

**admin pages**
- [x] /admin.etlua
- [x] /ip_admin.etlua
- [x] /carousel_admin.etlua
- [x] /totm.etlua
- [x] /user_admin.etlua
- [x] /users.etlua
- [x] /zombie_admin.etlua

**collection listings** 
- [x] /events.etlua # should be easy
- [x] /examples.etlua # should be easy
- [x] /explore.etlua

**teacher pages**
- [x] /bulk.etlua
- [x] /learners.etlua
- [x] /teacher.etlua

**users**
- [x] /change_email.etlua
- [x] /change_password.etlua
- [x] /followed_users.etlua
- [x] /followed.etlua
- [x] /forgot_password.etlua
- [x] /forgot_username.etlua
- [x] /login.etlua
- [x] /my_collections.etlua
- [x] /my_followers.etlua
- [x] /my_projects.etlua
- [x] /profile.etlua
- [x] /sign_up.etlua

**rest of the site**
- [x] /all_totms.etlua
- [x] /carousel_bs.etlua
- [x] /carousel.etlua
- [x] /collection.etlua
- [x] /collections.etlua
- [x] /delete_user.etlua
- [ ] /embed.etlua   # Do Last, will be more custom
- [x] /embedded.etlua # Do Last, will be more custom
- [x] /error.etlua
- [x] /flags.etlua
- [x] /footer.etlua
- [x] /grid_bs.etlua
- [x] /grid.etlua
- [x] /header.etlua
- [x] /index_bs.etlua
- [x] /index.etlua
- [x] /layout_bs.etlua
- [x] /layout.etlua
- [x] /localize.etlua
- [x] /message.etlua
- [x] /navigation_bar.etlua
- [x] /project.etlua
- [x] /search.etlua
- [x] /user.etlua

**Comment by cycomachead** [comment]
Ordering:

- collections carousel
- get all the basic carousel + grid pages done
- then do the projects page
- then do the collections page
- then do admin pages
- then investigate dialog + custom alert

- then delete pure css
- then delete the old lua files
- then rename layout
- then remove the `_bs` from all files

**Comment by cycomachead** [comment]
#398 will finish this all... just the collection page to go.
